examples/some-verifier/src/main.rs

```diff
@@ -0,0 +1,233 @@
+        )
+        .route(
+            "/verifications/discord/:id",
+            get(handle_get_verifications::<Discord>),
+        )
+        .with_state(state);
+
+    let socket = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), app.api_port);
+    axum::Server::bind(&socket)
+        .serve(router.into_make_service())
+        .await?;
+
+    Ok(())
+}
+
+async fn handle_get_verifications<P: DbPlatform>(
+    State(state): State<AppState>,
+    Path(id): Path<i64>,
+) -> Json<Vec<Verification>> {
+    let accounts = state.database.get_accounts::<P>(id).await;
+    match accounts {
+        Ok(accounts) => {
+            // Futures that simultaneously look up username and revocation status of user
+            // The futures are then mapped to Verifications
+            let futures = accounts.iter().map(|acc| {
+                future::try_join3(
+                    async { Ok(acc.platform) },
+                    get_username(&state, acc),
+                    state
+                        .database
+                        .get_revocation_status(acc)
+                        .map_err(|e| anyhow!(e)),
+                )
+                .map_ok(|(platform, username, revoked)| Verification {
+                    platform,
+                    username,
+                    revoked,
+                })
+            });
+
+            // Keep all the non-error values since the others could not get a username
+            // or a revocation status for whatever reason, probably because the user
+            // does not have that platform
+            let verifications = future::join_all(futures)
+                .await
+                .into_iter()
+                .filter_map(|res| res.ok())
+                .collect();
+            Json(verifications)
+        }
+        Err(_) => Json(vec![]),
+    }
+}
+
+/// Looks up the username of the given account.
+async fn get_username(state: &AppState, account: &Account) -> anyhow::Result<String> {
+    #[derive(Deserialize)]
+    struct DiscordUser {
+        username: String,
+        discriminator: String,
+    }
+
+    let id = account.id as u64;
+    match account.platform {
+        // TODO: This is unreliable if the user deletes their chat with the bot
+        Platform::Telegram => {
+            let user = state
+                .tdlib_client
+                .get_user(GetUser::builder().user_id(account.id).build())
+                .await?;
+            let name = if user.username().is_empty() {
+                if user.last_name().is_empty() {
+                    format!("{}", user.first_name())
+                } else {
+                    format!("{} {}", user.first_name(), user.last_name())
+                }
+            } else {
+                user.username().clone()
+            };
+            Ok(name)
+        }
+        Platform::Discord => {
+            const API_ENDPOINT: &'static str = "https://discord.com/api/v10";
```

Comment:
Define a top-level constant.

Reply:
Will be fixed, when #17 is eventually merged.